contracts/polygon/connectors/aave/v3-import/helpers.sol

```diff
@@ -275,7 +279,7 @@ contract AaveHelpers is Helper {
 						0,
 						0
 					);
-					if (!getIsColl(_token, address(this))) {
+					if (!getIsColl(_token, userAccount)) {
```

Comment:
@Richa-iitr why? We need to enable collateral on `address(this)`

Reply:
Got it, updated.